upup/pkg/fi/fitasks/keypair.go

```diff
@@ -173,13 +181,18 @@ func (_ *Keypair) Render(c *fi.Context, a, e, changes *Keypair) error {
 	createCertificate := false
 	if a == nil {
 		createCertificate = true
+		glog.V(8).Infof("creating brand new certificate")
 	} else if changes != nil {
+		glog.V(8).Infof("creating certificate as changes are not nil")
 		if changes.AlternateNames != nil {
 			createCertificate = true
+			glog.V(8).Infof("creating certificate new AlternateNames")
 		} else if changes.Subject != "" {
 			createCertificate = true
+			glog.V(8).Infof("creating certificate new Subject")
 		} else if changes.Type != "" {
 			createCertificate = true
+			glog.V(8).Infof("creating certificate new Type")
 		} else {
 			glog.Warningf("Ignoring changes in key: %v", fi.DebugAsJsonString(changes))
```

Comment:
@chrislovecnm As per our out-of-band convo, can we suppress this during an upgrade?

Reply:
Done